src/components/daily/bolustooltip/BolusTooltip.js

```diff
@@ -172,10 +173,48 @@ class BolusTooltip extends React.Component {
     return extendedLine;
   }
 
+  getFormatedInputTime(bolus, timePrefs) {
+    const time = formatLocalizedFromUTC(bolus.inputTime, timePrefs, getHourMinuteFormat());
+    return time;
+  }
+
+  getBolusTypeLine(bolusType) {
+    return bolusType && (
+      <div className={styles.bolus}>
+        <div className={styles.label}>{t('Bolus')}</div>
+        <div className={styles.value}>{t(bolusType)}</div>
+      </div>
+      );
+  }
+
+  getIobLine(iob) {
+    return !!iob && (
+      <div className={styles.iob}>
+        <div className={styles.label}>{t('IOB')}</div>
+        <div className={styles.value}>{`${formatInsulin(iob)}`}</div>
```

Comment:
```suggestion
        <div className={styles.value}>{formatInsulin(iob)}</div>
```
`formatInsulin()` function already return a string.